client/my-sites/patterns/components/pattern-preview/style.scss

```
@@ -33,3 +33,21 @@
 		display: none;
 	}
 }
+
+.pattern-preview__resizer .components-resizable-box__handle-right {
+	&::after {
+		background-color: #a7aaad;
+		border-radius: 3px;
+		box-shadow: none;
+		height: 50px;
+		margin-right: -10px;
+		right: calc(50% - 5px);
```

```suggestion
		right: calc(50% - 5px);
		top: calc(50% - 8px - 3rem);
```
The value I've used here isn't ideal, but the idea is to center the handle to the actual preview viewport, as opposed to centering it to the entire `.pattern-preview` (which also includes the title).